scripts/qesap/lib/cmds.py

```diff
@@ -176,10 +176,10 @@ def cmd_terraform(configure_data, base_project, dryrun, workspace='default', des
     cmds = []
     terraform_common_cmd = [config.get_terraform_bin(), f"-chdir={cfg_paths['provider']}"]
     if destroy:
+        cmds.append(terraform_common_cmd + ['destroy', '-auto-approve', '-no-color'])
```

Comment:
The deployment has to be destroyed before to destroy the workspace, otherwise Terraform refuse to destroy the workspace saying that there are some resources associated to it.